VMEncryption/main/oscrypto/rhel_72_lvm/encryptstates/PrereqState.py

```diff
@@ -19,8 +19,9 @@
 # Requires Python 2.7+
 #
 
-from OSEncryptionState import *
-from pprint import pprint
+from OSEncryptionState import OSEncryptionState
+import re
```

Comment:
Why import re ?  don't see it called in any of the new code shown in the diff

Reply:
Some pre-existing code uses the re module.
It was getting indirectly imported earlier (some code in OSEncryptionState imported it) which is not a good way to import modules. In general I've been trying to remove "*" imports from the files I touch to prevent exactly this.